2022-01-05 00:01:43,010 INFO client.py "Имя клиента" test3
2022-01-05 00:01:43,011 INFO client.py Установлен стандартный порт и адрес
2022-01-05 00:01:43,061 INFO client.py "Имя клиента" test1
2022-01-05 00:01:43,062 INFO client.py Установлен стандартный порт и адрес
2022-01-05 00:01:43,066 INFO client.py "Имя клиента" test2
2022-01-05 00:01:43,066 INFO client.py Установлен стандартный порт и адрес
2022-01-05 00:01:43,084 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:01:43,085 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: test3
2022-01-05 00:01:43,089 DEBUG client.py Сформировано PRESENCE сообщение для пользователя test3
2022-01-05 00:01:43,093 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('test3',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:01:43,095 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=432, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60346), raddr=('127.0.0.1', 7777)>, {'ACTION': 'PRESENCE', 'TIME': 1641330103.0894876, 'USER': {'ACCOUNT_NAME': 'test3'}}), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:01:43,106 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:01:43,106 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: test1
2022-01-05 00:01:43,113 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:01:43,114 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: test2
2022-01-05 00:01:43,119 DEBUG client.py Сформировано PRESENCE сообщение для пользователя test1
2022-01-05 00:01:43,121 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('test1',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:01:43,123 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=432, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60346), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:01:43,123 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 200}
2022-01-05 00:01:43,123 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=452, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60347), raddr=('127.0.0.1', 7777)>, {'ACTION': 'PRESENCE', 'TIME': 1641330103.1194904, 'USER': {'ACCOUNT_NAME': 'test1'}}), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:01:43,124 DEBUG client.py Была вызвана функция process_response_ans
 c параметрами ({'RESPONSE': 200},), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:01:43,126 DEBUG client.py Сформировано PRESENCE сообщение для пользователя test2
2022-01-05 00:01:43,128 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('test2',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:01:43,130 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=452, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60347), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:01:43,130 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 200}
2022-01-05 00:01:43,132 DEBUG client.py Была вызвана функция process_response_ans
 c параметрами ({'RESPONSE': 200},), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:01:43,137 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60348), raddr=('127.0.0.1', 7777)>, {'ACTION': 'PRESENCE', 'TIME': 1641330103.1264913, 'USER': {'ACCOUNT_NAME': 'test2'}}), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:01:43,142 DEBUG client.py Запущены процессы
2022-01-05 00:01:43,146 DEBUG client.py Запущены процессы
2022-01-05 00:01:43,643 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60348), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:01:43,643 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 200}
2022-01-05 00:01:43,644 DEBUG client.py Была вызвана функция process_response_ans
 c параметрами ({'RESPONSE': 200},), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:01:43,647 DEBUG client.py Запущены процессы
2022-01-05 00:01:56,351 DEBUG client.py Сформирован словарь сообщения: {'ACTION': 'MESSAGE', 'SENDER': 'test3', 'DESTINATION': 'test1', 'TIME': 1641330116.3504837, 'MESSAGE_TEXT': 'fsdfsdfsdfsdfsdf'}
2022-01-05 00:01:56,353 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=432, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60346), raddr=('127.0.0.1', 7777)>, {'ACTION': 'MESSAGE', 'SENDER': 'test3', 'DESTINATION': 'test1', 'TIME': 1641330116.3504837, 'MESSAGE_TEXT': 'fsdfsdfsdfsdfsdf'}), {}.Вызов из модуля common.settings.
 Вызов изфункции self._bootstrap_inner().
 Вызов из функции create_message
2022-01-05 00:01:56,354 INFO client.py Отправлено сообщение для пользователя test1
2022-01-05 00:01:56,355 DEBUG client.py Была вызвана функция create_message
 c параметрами (<User_interface(Thread-2, started daemon 10376)>, <socket.socket fd=432, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60346), raddr=('127.0.0.1', 7777)>, 'test3'), {}.Вызов из модуля __main__.
 Вызов изфункции self._bootstrap_inner().
 Вызов из функции run
2022-01-05 00:01:56,406 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=452, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60347), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции self._bootstrap_inner().
 Вызов из функции run
2022-01-05 00:01:56,407 INFO client.py Получено сообщение от пользователя test3:
fsdfsdfsdfsdfsdf
2022-01-05 00:46:49,927 INFO client.py Установлен стандартный порт и адрес
2022-01-05 00:46:49,957 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:46:49,957 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: default
2022-01-05 00:46:52,006 ERROR client.py [WinError 10061] Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение
2022-01-05 00:47:03,903 INFO client.py Установлен стандартный порт и адрес
2022-01-05 00:47:03,937 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:47:03,937 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: default
2022-01-05 00:47:03,938 DEBUG client.py Сформировано PRESENCE сообщение для пользователя default
2022-01-05 00:47:03,940 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('default',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:47:03,942 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=356, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60391), raddr=('127.0.0.1', 7777)>, {'ACTION': 'PRESENCE', 'TIME': 1641332823.9385035, 'USER': {'ACCOUNT_NAME': 'default'}}), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:47:04,446 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=356, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60391), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:47:04,446 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 200}
2022-01-05 00:47:04,447 DEBUG client.py Была вызвана функция process_response_ans
 c параметрами ({'RESPONSE': 200},), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:47:04,449 DEBUG client.py Запущены процессы
2022-01-05 00:47:08,607 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=356, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60391), raddr=('127.0.0.1', 7777)>, {'ACTION': 'EXIT', 'TIME': 1641332828.6044967, 'SENDER': 'default'}), {}.Вызов из модуля common.settings.
 Вызов изфункции self._bootstrap_inner().
 Вызов из функции run
2022-01-05 00:47:08,607 INFO client.py Завершение работы по команде пользователя.
2022-01-05 00:47:09,040 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=356, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60391), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции self._bootstrap_inner().
 Вызов из функции run
2022-01-05 00:47:09,040 ERROR client.py Получено некорректное сообщение с сервера: {'RESPONSE': 400, 'ERROR': 'Запрос некорректен.'}
2022-01-05 00:47:09,481 DEBUG client.py Была вызвана функция main
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции <module>
2022-01-05 00:47:19,626 INFO client.py "Имя клиента" test1
2022-01-05 00:47:19,627 INFO client.py Установлен стандартный порт и адрес
2022-01-05 00:47:19,627 INFO client.py "Имя клиента" test2
2022-01-05 00:47:19,628 INFO client.py Установлен стандартный порт и адрес
2022-01-05 00:47:19,689 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:47:19,689 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: test1
2022-01-05 00:47:19,694 DEBUG client.py Сформировано PRESENCE сообщение для пользователя test1
2022-01-05 00:47:19,696 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:47:19,696 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: test2
2022-01-05 00:47:19,696 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('test1',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:47:19,700 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=388, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60392), raddr=('127.0.0.1', 7777)>, {'ACTION': 'PRESENCE', 'TIME': 1641332839.6944978, 'USER': {'ACCOUNT_NAME': 'test1'}}), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:47:19,701 DEBUG client.py Сформировано PRESENCE сообщение для пользователя test2
2022-01-05 00:47:19,703 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('test2',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:47:19,706 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=388, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60392), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
tings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:47:19,706 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 200}
2022-01-05 00:47:19,708 DEBUG client.py Была вызвана функция process_response_ans
 c параметрами ({'RESPONSE': 200},), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:47:19,718 DEBUG client.py Запущены процессы
2022-01-05 00:47:19,745 INFO client.py "Имя клиента" test3
2022-01-05 00:47:19,746 INFO client.py Установлен стандартный порт и адрес
2022-01-05 00:47:19,819 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:47:19,819 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: test3
2022-01-05 00:47:19,824 DEBUG client.py Сформировано PRESENCE сообщение для пользователя test3
2022-01-05 00:47:19,829 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=448, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60393), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:47:19,829 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 200}
2022-01-05 00:47:19,831 DEBUG client.py Была вызвана функция process_response_ans
 c параметрами ({'RESPONSE': 200},), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:47:19,834 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('test3',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:47:19,838 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=348, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60394), raddr=('127.0.0.1', 7777)>, {'ACTION': 'PRESENCE', 'TIME': 1641332839.824499, 'USER': {'ACCOUNT_NAME': 'test3'}}), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:47:19,843 DEBUG client.py Запущены процессы
2022-01-05 00:47:20,351 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=348, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60394), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:47:20,351 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 200}
2022-01-05 00:47:20,353 DEBUG client.py Была вызвана функция process_response_ans
 c параметрами ({'RESPONSE': 200},), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:47:20,355 DEBUG client.py Запущены процессы
2022-01-05 00:47:39,233 DEBUG client.py Сформирован словарь сообщения: {'ACTION': 'MESSAGE', 'SENDER': 'test1', 'DESTINATION': 'test3', 'TIME': 1641332859.2334952, 'MESSAGE_TEXT': 'dsfsdf'}
2022-01-05 00:47:39,237 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=388, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60392), raddr=('127.0.0.1', 7777)>, {'ACTION': 'MESSAGE', 'SENDER': 'test1', 'DESTINATION': 'test3', 'TIME': 1641332859.2334952, 'MESSAGE_TEXT': 'dsfsdf'}), {}.Вызов из модуля common.settings.
 Вызов изфункции self._bootstrap_inner().
 Вызов из функции create_message
2022-01-05 00:47:39,237 INFO client.py Отправлено сообщение для пользователя test3
2022-01-05 00:47:39,239 DEBUG client.py Была вызвана функция create_message
 c параметрами (<Interface(Thread-2, started daemon 10488)>, <socket.socket fd=388, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60392), raddr=('127.0.0.1', 7777)>, 'test1'), {}.Вызов из модуля __main__.
 Вызов изфункции self._bootstrap_inner().
 Вызов из функции run
2022-01-05 00:47:39,249 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=348, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60394), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции self._bootstrap_inner().
 Вызов из функции run
2022-01-05 00:47:39,250 INFO client.py Получено сообщение от пользователя test1:
dsfsdf
2022-01-05 00:48:25,946 INFO client.py "Имя клиента" test2
2022-01-05 00:48:25,947 INFO client.py Установлен стандартный порт и адрес
2022-01-05 00:48:25,951 INFO client.py "Имя клиента" test3
2022-01-05 00:48:25,951 INFO client.py Установлен стандартный порт и адрес
2022-01-05 00:48:25,986 INFO client.py "Имя клиента" test1
2022-01-05 00:48:25,986 INFO client.py Установлен стандартный порт и адрес
2022-01-05 00:48:26,004 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:26,004 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: test3
2022-01-05 00:48:26,006 DEBUG client.py Сформировано PRESENCE сообщение для пользователя test3
2022-01-05 00:48:26,014 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:26,014 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: test2
2022-01-05 00:48:26,019 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('test3',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:26,020 DEBUG client.py Сформировано PRESENCE сообщение для пользователя test2
2022-01-05 00:48:26,022 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('test2',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:26,024 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=428, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60396), raddr=('127.0.0.1', 7777)>, {'ACTION': 'PRESENCE', 'TIME': 1641332906.0204988, 'USER': {'ACCOUNT_NAME': 'test2'}}), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:26,025 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=300, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60395), raddr=('127.0.0.1', 7777)>, {'ACTION': 'PRESENCE', 'TIME': 1641332906.0064983, 'USER': {'ACCOUNT_NAME': 'test3'}}), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:26,027 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=300, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60395), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:26,029 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 400, 'ERROR': 'Имя пользователя уже занято.'}
2022-01-05 00:48:26,031 DEBUG client.py Была вызвана функция process_response_ans
 c параметрами ({'RESPONSE': 400, 'ERROR': 'Имя пользователя уже занято.'},), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:26,035 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:26,035 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: test1
2022-01-05 00:48:26,035 ERROR client.py Критическая ошибка
2022-01-05 00:48:26,036 DEBUG client.py Запущены процессы
2022-01-05 00:48:26,038 DEBUG client.py Сформировано PRESENCE сообщение для пользователя test1
2022-01-05 00:48:26,041 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('test1',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:26,043 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=428, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60396), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:26,043 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 400, 'ERROR': 'Имя пользователя уже занято.'}
2022-01-05 00:48:26,045 DEBUG client.py Была вызвана функция process_response_ans
 c параметрами ({'RESPONSE': 400, 'ERROR': 'Имя пользователя уже занято.'},), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:26,047 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=420, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60397), raddr=('127.0.0.1', 7777)>, {'ACTION': 'PRESENCE', 'TIME': 1641332906.0384986, 'USER': {'ACCOUNT_NAME': 'test1'}}), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:26,048 ERROR client.py Критическая ошибка
2022-01-05 00:48:26,049 DEBUG client.py Запущены процессы
2022-01-05 00:48:26,556 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=420, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60397), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:26,556 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 400, 'ERROR': 'Имя пользователя уже занято.'}
2022-01-05 00:48:26,559 DEBUG client.py Была вызвана функция process_response_ans
 c параметрами ({'RESPONSE': 400, 'ERROR': 'Имя пользователя уже занято.'},), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:26,561 ERROR client.py Критическая ошибка
2022-01-05 00:48:26,561 DEBUG client.py Запущены процессы
2022-01-05 00:48:27,049 DEBUG client.py Была вызвана функция main
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции <module>
2022-01-05 00:48:27,065 DEBUG client.py Была вызвана функция main
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции <module>
2022-01-05 00:48:27,575 DEBUG client.py Была вызвана функция main
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции <module>
2022-01-05 00:48:37,672 INFO client.py "Имя клиента" test1
2022-01-05 00:48:37,673 INFO client.py Установлен стандартный порт и адрес
2022-01-05 00:48:37,700 INFO client.py "Имя клиента" test2
2022-01-05 00:48:37,700 INFO client.py Установлен стандартный порт и адрес
2022-01-05 00:48:37,721 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:37,721 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: test1
2022-01-05 00:48:37,725 DEBUG client.py Сформировано PRESENCE сообщение для пользователя test1
2022-01-05 00:48:37,728 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('test1',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:37,730 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=300, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60398), raddr=('127.0.0.1', 7777)>, {'ACTION': 'PRESENCE', 'TIME': 1641332917.7254996, 'USER': {'ACCOUNT_NAME': 'test1'}}), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:37,748 INFO client.py "Имя клиента" test3
2022-01-05 00:48:37,748 INFO client.py Установлен стандартный порт и адрес
2022-01-05 00:48:37,750 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:37,750 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: test2
2022-01-05 00:48:37,756 DEBUG client.py Сформировано PRESENCE сообщение для пользователя test2
2022-01-05 00:48:37,758 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('test2',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:37,760 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=300, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60398), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:37,760 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=296, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60399), raddr=('127.0.0.1', 7777)>, {'ACTION': 'PRESENCE', 'TIME': 1641332917.7564983, 'USER': {'ACCOUNT_NAME': 'test2'}}), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:37,760 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 200}
2022-01-05 00:48:37,761 DEBUG client.py Была вызвана функция process_response_ans
 c параметрами ({'RESPONSE': 200},), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:37,765 DEBUG client.py Запущены процессы
2022-01-05 00:48:37,796 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:37,797 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: test3
2022-01-05 00:48:37,799 DEBUG client.py Сформировано PRESENCE сообщение для пользователя test3
2022-01-05 00:48:37,801 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('test3',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:37,803 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=456, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60400), raddr=('127.0.0.1', 7777)>, {'ACTION': 'PRESENCE', 'TIME': 1641332917.7994962, 'USER': {'ACCOUNT_NAME': 'test3'}}), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:37,803 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=296, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60399), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:37,804 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 200}
2022-01-05 00:48:37,805 DEBUG client.py Была вызвана функция process_response_ans
 c параметрами ({'RESPONSE': 200},), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:37,808 DEBUG client.py Запущены процессы
2022-01-05 00:48:38,311 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=456, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60400), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:38,311 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 200}
2022-01-05 00:48:38,313 DEBUG client.py Была вызвана функция process_response_ans
 c параметрами ({'RESPONSE': 200},), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:38,316 DEBUG client.py Запущены процессы
2022-01-06 01:12:18,227 INFO client.py "Имя клиента" test1
2022-01-06 01:12:18,227 INFO client.py Установлен стандартный порт и адрес
2022-01-06 01:12:18,268 INFO client.py "Имя клиента" test2
2022-01-06 01:12:18,269 INFO client.py Установлен стандартный порт и адрес
2022-01-06 01:12:18,270 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-06 01:12:18,270 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: test1
2022-01-06 01:12:18,276 DEBUG client.py Сформировано PRESENCE сообщение для пользователя test1
2022-01-06 01:12:18,277 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('test1',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-06 01:12:18,279 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=484, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 63320), raddr=('127.0.0.1', 7777)>, {'ACTION': 'PRESENCE', 'TIME': 1641420738.276918, 'USER': {'ACCOUNT_NAME': 'test1'}}), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-06 01:12:18,304 INFO client.py "Имя клиента" test3
2022-01-06 01:12:18,304 INFO client.py Установлен стандартный порт и адрес
2022-01-06 01:12:18,305 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-06 01:12:18,305 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: test2
2022-01-06 01:12:18,308 DEBUG client.py Сформировано PRESENCE сообщение для пользователя test2
2022-01-06 01:12:18,309 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('test2',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-06 01:12:18,312 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=292, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 63321), raddr=('127.0.0.1', 7777)>, {'ACTION': 'PRESENCE', 'TIME': 1641420738.3079169, 'USER': {'ACCOUNT_NAME': 'test2'}}), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-06 01:12:18,372 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=484, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 63320), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-06 01:12:18,373 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 200}
2022-01-06 01:12:18,374 DEBUG client.py Была вызвана функция process_response_ans
 c параметрами ({'RESPONSE': 200},), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-06 01:12:18,380 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-06 01:12:18,380 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: test3
2022-01-06 01:12:18,389 DEBUG client.py Сформировано PRESENCE сообщение для пользователя test3
2022-01-06 01:12:18,392 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('test3',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-06 01:12:18,392 DEBUG client.py Запущены процессы
2022-01-06 01:12:18,395 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=452, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 63322), raddr=('127.0.0.1', 7777)>, {'ACTION': 'PRESENCE', 'TIME': 1641420738.3899171, 'USER': {'ACCOUNT_NAME': 'test3'}}), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-06 01:12:18,437 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=292, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 63321), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-06 01:12:18,437 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 200}
2022-01-06 01:12:18,439 DEBUG client.py Была вызвана функция process_response_ans
 c параметрами ({'RESPONSE': 200},), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-06 01:12:18,455 DEBUG client.py Запущены процессы
2022-01-06 01:12:19,010 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=452, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 63322), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-06 01:12:19,010 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 200}
2022-01-06 01:12:19,011 DEBUG client.py Была вызвана функция process_response_ans
 c параметрами ({'RESPONSE': 200},), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-06 01:12:19,014 DEBUG client.py Запущены процессы
2022-01-06 01:14:21,374 ERROR client.py Критическая ошибка
2022-01-06 01:14:21,374 ERROR client.py Критическая ошибка
2022-01-06 01:14:21,399 DEBUG client.py Была вызвана функция main
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции <module>
2022-01-06 01:14:21,526 DEBUG client.py Была вызвана функция main
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции <module>
2022-01-06 01:14:21,927 DEBUG client.py Была вызвана функция main
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции <module>
2022-01-06 01:14:27,070 INFO client.py "Имя клиента" test1
2022-01-06 01:14:27,071 INFO client.py Установлен стандартный порт и адрес
2022-01-06 01:14:27,083 INFO client.py "Имя клиента" test2
2022-01-06 01:14:27,083 INFO client.py Установлен стандартный порт и адрес
2022-01-06 01:14:27,098 INFO client.py "Имя клиента" test3
2022-01-06 01:14:27,098 INFO client.py Установлен стандартный порт и адрес
2022-01-06 01:14:27,110 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-06 01:14:27,111 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: test1
2022-01-06 01:14:27,114 DEBUG client.py Сформировано PRESENCE сообщение для пользователя test1
2022-01-06 01:14:27,116 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('test1',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-06 01:14:27,117 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=380, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 63325), raddr=('127.0.0.1', 7777)>, {'ACTION': 'PRESENCE', 'TIME': 1641420867.1149144, 'USER': {'ACCOUNT_NAME': 'test1'}}), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-06 01:14:27,125 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-06 01:14:27,125 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: test2
2022-01-06 01:14:27,127 DEBUG client.py Сформировано PRESENCE сообщение для пользователя test2
2022-01-06 01:14:27,147 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('test2',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-06 01:14:27,150 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=392, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 63326), raddr=('127.0.0.1', 7777)>, {'ACTION': 'PRESENCE', 'TIME': 1641420867.1279147, 'USER': {'ACCOUNT_NAME': 'test2'}}), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-06 01:14:27,152 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-06 01:14:27,152 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: test3
2022-01-06 01:14:27,163 DEBUG client.py Сформировано PRESENCE сообщение для пользователя test3
2022-01-06 01:14:27,167 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('test3',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-06 01:14:27,168 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=380, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 63325), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-06 01:14:27,168 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 200}
2022-01-06 01:14:27,169 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=480, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 63327), raddr=('127.0.0.1', 7777)>, {'ACTION': 'PRESENCE', 'TIME': 1641420867.1639168, 'USER': {'ACCOUNT_NAME': 'test3'}}), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-06 01:14:27,170 DEBUG client.py Была вызвана функция process_response_ans
 c параметрами ({'RESPONSE': 200},), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-06 01:14:27,181 DEBUG client.py Запущены процессы
2022-01-06 01:14:27,197 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=392, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 63326), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-06 01:14:27,197 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 200}
2022-01-06 01:14:27,199 DEBUG client.py Была вызвана функция process_response_ans
 c параметрами ({'RESPONSE': 200},), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-06 01:14:27,202 DEBUG client.py Запущены процессы
2022-01-06 01:14:27,231 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=480, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 63327), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-06 01:14:27,231 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 200}
2022-01-06 01:14:27,233 DEBUG client.py Была вызвана функция process_response_ans
 c параметрами ({'RESPONSE': 200},), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-06 01:14:27,235 DEBUG client.py Запущены процессы
2022-01-06 01:14:43,652 DEBUG client.py Сформирован словарь сообщения: {'ACTION': 'MESSAGE', 'SENDER': 'test2', 'DESTINATION': 'test3', 'TIME': 1641420883.6519094, 'MESSAGE_TEXT': 'gfdtfddytr'}
2022-01-06 01:14:43,666 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=392, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 63326), raddr=('127.0.0.1', 7777)>, {'ACTION': 'MESSAGE', 'SENDER': 'test2', 'DESTINATION': 'test3', 'TIME': 1641420883.6519094, 'MESSAGE_TEXT': 'gfdtfddytr'}), {}.Вызов из модуля common.settings.
 Вызов изфункции self._bootstrap_inner().
 Вызов из функции create_message
2022-01-06 01:14:43,666 INFO client.py Отправлено сообщение для пользователя test3
2022-01-06 01:14:43,668 DEBUG client.py Была вызвана функция create_message
 c параметрами (<Interface(Thread-2, started daemon 4112)>, <socket.socket fd=392, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 63326), raddr=('127.0.0.1', 7777)>, 'test2'), {}.Вызов из модуля __main__.
 Вызов изфункции self._bootstrap_inner().
 Вызов из функции run
2022-01-06 01:14:44,117 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=480, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 63327), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции self._bootstrap_inner().
 Вызов из функции run
2022-01-06 01:14:44,118 INFO client.py Получено сообщение от пользователя test2:
gfdtfddytr
2022-01-06 01:14:49,253 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=392, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 63326), raddr=('127.0.0.1', 7777)>, {'ACTION': 'EXIT', 'TIME': 1641420889.251909, 'SENDER': 'test2'}), {}.Вызов из модуля common.settings.
 Вызов изфункции self._bootstrap_inner().
 Вызов из функции run
2022-01-06 01:14:49,253 INFO client.py Завершение работы по команде пользователя.
2022-01-06 01:14:49,745 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=392, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 63326), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции self._bootstrap_inner().
 Вызов из функции run
2022-01-06 01:14:49,745 ERROR client.py Получено некорректное сообщение с сервера: {'RESPONSE': 400, 'ERROR': 'Запрос некорректен.'}
2022-01-06 01:14:50,382 DEBUG client.py Была вызвана функция main
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции <module>
2022-01-06 01:15:31,880 ERROR client.py Критическая ошибка
2022-01-06 01:15:31,880 ERROR client.py Критическая ошибка
2022-01-06 01:15:32,640 DEBUG client.py Была вызвана функция main
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции <module>
2022-01-06 01:15:32,687 DEBUG client.py Была вызвана функция main
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции <module>
2022-01-09 20:40:28,313 INFO client.py "Имя клиента" test2
2022-01-09 20:40:28,313 INFO client.py Установлен стандартный порт и адрес
2022-01-09 20:40:28,313 INFO client.py Установлен стандартный порт и адрес
2022-01-09 20:40:28,313 INFO client.py "Имя клиента" test3
2022-01-09 20:40:28,313 INFO client.py Установлен стандартный порт и адрес
2022-01-09 20:40:28,342 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 20:40:28,342 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: test1
2022-01-09 20:40:28,343 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 20:40:28,343 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: test3
2022-01-09 20:40:28,343 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 20:40:28,343 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: test2
2022-01-09 20:40:29,872 DEBUG client.py Сформировано PRESENCE сообщение для пользователя test1
2022-01-09 20:40:29,874 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('test1',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 20:40:29,874 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('test3',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 20:40:29,874 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('test2',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 20:40:29,875 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=424, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 50090), raddr=('127.0.0.1', 7777)>, {'ACTION': 'PRESENCE', 'TIME': 1641750029.87237, 'USER': {'ACCOUNT_NAME': 'test1'}}), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 20:40:29,875 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=440, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 50092), raddr=('127.0.0.1', 7777)>, {'ACTION': 'PRESENCE', 'TIME': 1641750029.87237, 'USER': {'ACCOUNT_NAME': 'test2'}}), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 20:40:29,929 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=452, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 50091), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 20:40:29,929 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 200}
2022-01-09 20:40:29,930 DEBUG client.py Была вызвана функция process_response_ans
 c параметрами ({'RESPONSE': 200},), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 20:40:29,932 DEBUG client.py Запущены процессы
2022-01-09 20:40:29,961 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=424, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 50090), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 20:40:29,961 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 200}
2022-01-09 20:40:29,963 DEBUG client.py Была вызвана функция process_response_ans
 c параметрами ({'RESPONSE': 200},), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 20:40:29,964 DEBUG client.py Запущены процессы
2022-01-09 20:40:29,993 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=440, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 50092), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 20:40:29,993 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 200}
2022-01-09 20:40:29,994 DEBUG client.py Была вызвана функция process_response_ans
 c параметрами ({'RESPONSE': 200},), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 20:40:29,996 DEBUG client.py Запущены процессы
2022-01-09 20:40:43,764 DEBUG client.py Сформирован словарь сообщения: {'ACTION': 'MESSAGE', 'SENDER': 'test3', 'DESTINATION': 'ihyui', 'TIME': 1641750043.7648258, 'MESSAGE_TEXT': 'ip'}
2022-01-09 20:40:43,767 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=452, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 50091), raddr=('127.0.0.1', 7777)>, {'ACTION': 'MESSAGE', 'SENDER': 'test3', 'DESTINATION': 'ihyui', 'TIME': 1641750043.7648258, 'MESSAGE_TEXT': 'ip'}), {}.Вызов из модуля common.settings.
 Вызов изфункции self._bootstrap_inner().
 Вызов из функции create_message
2022-01-09 20:40:43,767 INFO client.py Отправлено сообщение для пользователя ihyui
2022-01-09 20:40:43,769 DEBUG client.py Была вызвана функция create_message
 c параметрами (<Interface(Thread-2, started daemon 8452)>, <socket.socket fd=452, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 50091), raddr=('127.0.0.1', 7777)>, 'test3'), {}.Вызов из модуля __main__.
 Вызов изфункции self._bootstrap_inner().
 Вызов из функции run
2022-01-09 20:40:59,697 ERROR client.py Критическая ошибка
2022-01-09 20:41:00,162 DEBUG client.py Была вызвана функция main
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции <module>
2022-01-09 20:41:00,194 DEBUG client.py Была вызвана функция main
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции <module>
2022-01-09 20:54:05,899 INFO client.py "Имя клиента" test2
2022-01-09 20:54:05,899 INFO client.py Установлен стандартный порт и адрес
2022-01-09 20:54:05,901 INFO client.py "Имя клиента" test3
2022-01-09 20:54:05,901 INFO client.py Установлен стандартный порт и адрес
2022-01-09 20:54:05,905 INFO client.py "Имя клиента" test1
2022-01-09 20:54:05,905 INFO client.py Установлен стандартный порт и адрес
2022-01-09 20:54:05,985 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 20:54:05,985 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: test2
2022-01-09 20:54:05,986 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 20:54:05,986 ERROR client.py 'module' object is not callable
2022-01-09 20:54:05,986 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: test3
2022-01-09 20:54:05,987 ERROR client.py 'module' object is not callable
2022-01-09 20:54:05,990 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 20:54:05,990 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: test1
2022-01-09 20:54:05,991 ERROR client.py 'module' object is not callable
2022-01-09 20:54:17,195 INFO client.py Установлен стандартный порт и адрес
2022-01-09 20:54:17,277 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 20:54:17,277 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: default
2022-01-09 20:54:17,277 ERROR client.py 'module' object is not callable
2022-01-09 20:59:57,620 DEBUG client.py Была вызвана функция arg_parser
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 21:00:00,286 INFO client.py Запущен клиент с параметрами: адрес сервера: 127.0.0.1 , порт: 7777, имя пользователя: sdgfr
2022-01-09 21:00:00,287 DEBUG client.py Сформировано presence сообщение для пользователя sdgfr
2022-01-09 21:00:00,288 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('sdgfr',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 21:00:51,863 DEBUG client.py Была вызвана функция arg_parser
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 21:00:51,864 INFO client.py Запущен клиент с параметрами: адрес сервера: 127.0.0.1 , порт: 7777, имя пользователя: test1
2022-01-09 21:00:51,867 DEBUG client.py Была вызвана функция arg_parser
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 21:00:51,868 INFO client.py Запущен клиент с параметрами: адрес сервера: 127.0.0.1 , порт: 7777, имя пользователя: test2
2022-01-09 21:00:51,868 DEBUG client.py Была вызвана функция arg_parser
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 21:00:51,869 INFO client.py Запущен клиент с параметрами: адрес сервера: 127.0.0.1 , порт: 7777, имя пользователя: test3
2022-01-09 21:00:52,385 DEBUG client.py Сформировано presence сообщение для пользователя test3
2022-01-09 21:00:52,386 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('test3',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 21:00:52,387 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=380, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 50116), raddr=('127.0.0.1', 7777)>, {'action': 'presence', 'time': 1641751252.3859324, 'user': {'account_name': 'test3'}}), {}.Вызов из модуля common.utils.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 21:00:52,881 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=380, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 50116), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.utils.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 21:00:52,882 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 400, 'ERROR': 'Запрос некорректен.'}
2022-01-09 21:00:52,882 ERROR client.py В ответе сервера отсутствует необходимое поле response
2022-01-09 21:01:08,415 DEBUG client.py Была вызвана функция arg_parser
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 21:01:08,416 INFO client.py Запущен клиент с параметрами: адрес сервера: 127.0.0.1 , порт: 7777, имя пользователя: test3
2022-01-09 21:01:08,420 DEBUG client.py Была вызвана функция arg_parser
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 21:01:08,420 DEBUG client.py Была вызвана функция arg_parser
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 21:01:08,421 INFO client.py Запущен клиент с параметрами: адрес сервера: 127.0.0.1 , порт: 7777, имя пользователя: test2
2022-01-09 21:01:08,421 INFO client.py Запущен клиент с параметрами: адрес сервера: 127.0.0.1 , порт: 7777, имя пользователя: test1
2022-01-09 21:01:08,930 DEBUG client.py Сформировано presence сообщение для пользователя test3
2022-01-09 21:01:08,930 DEBUG client.py Сформировано presence сообщение для пользователя test2
2022-01-09 21:01:08,931 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('test1',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 21:01:08,931 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('test2',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 21:01:08,932 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('test3',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 21:01:08,933 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=644, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 50119), raddr=('127.0.0.1', 7777)>, {'action': 'presence', 'time': 1641751268.9305604, 'user': {'account_name': 'test1'}}), {}.Вызов из модуля common.utils.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 21:01:08,933 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=512, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 50118), raddr=('127.0.0.1', 7777)>, {'action': 'presence', 'time': 1641751268.9305604, 'user': {'account_name': 'test2'}}), {}.Вызов из модуля common.utils.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 21:01:09,267 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=476, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 50117), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.utils.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 21:01:09,268 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 400, 'ERROR': 'Запрос некорректен.'}
2022-01-09 21:01:09,268 ERROR client.py В ответе сервера отсутствует необходимое поле response
2022-01-09 21:01:09,287 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=644, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 50119), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.utils.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 21:01:09,288 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 400, 'ERROR': 'Запрос некорректен.'}
2022-01-09 21:01:09,288 ERROR client.py В ответе сервера отсутствует необходимое поле response
2022-01-09 21:01:09,296 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=512, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 50118), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.utils.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 21:01:09,296 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 400, 'ERROR': 'Запрос некорректен.'}
2022-01-09 21:01:09,296 ERROR client.py В ответе сервера отсутствует необходимое поле response
2022-01-09 21:01:56,909 DEBUG client.py Была вызвана функция arg_parser
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 21:02:00,846 INFO client.py Запущен клиент с параметрами: адрес сервера: 127.0.0.1 , порт: 7777, имя пользователя: test
2022-01-09 21:02:00,847 DEBUG client.py Сформировано presence сообщение для пользователя test
2022-01-09 21:02:00,848 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('test',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 21:02:00,849 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=400, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 50120), raddr=('127.0.0.1', 7777)>, {'action': 'presence', 'time': 1641751320.8477454, 'user': {'account_name': 'test'}}), {}.Вызов из модуля common.utils.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 21:02:01,357 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=400, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 50120), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.utils.
 Вызов изфункции main().
 Вызов из функции main
2022-01-09 21:02:01,357 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 400, 'ERROR': 'Запрос некорректен.'}
2022-01-09 21:02:01,358 ERROR client.py В ответе сервера отсутствует необходимое поле response
